<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>790</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::PreventContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>전원관리프로그램</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>859</x>
      <y>19</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btn_SA">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>전체선택</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>539</y>
      <width>1031</width>
      <height>213</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="SDL">
     <item>
      <widget class="QPushButton" name="btn_SD">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>72</pointsize>
        </font>
       </property>
       <property name="text">
        <string>동작</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>109</y>
      <width>1021</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="L1">
       <item>
        <widget class="QCheckBox" name="CB_1">
         <property name="text">
          <string>1번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_2">
         <property name="text">
          <string>2번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_3">
         <property name="text">
          <string>3번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_4">
         <property name="text">
          <string>4번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_5">
         <property name="text">
          <string>5번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_6">
         <property name="text">
          <string>6번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_7">
         <property name="text">
          <string>7번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_8">
         <property name="text">
          <string>8번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_9">
         <property name="text">
          <string>9번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_10">
         <property name="text">
          <string>10번 PC</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="L2">
       <item>
        <widget class="QCheckBox" name="CB_11">
         <property name="text">
          <string>11번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_12">
         <property name="text">
          <string>12번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_13">
         <property name="text">
          <string>13번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_14">
         <property name="text">
          <string>14번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_15">
         <property name="text">
          <string>15번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_16">
         <property name="text">
          <string>16번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_17">
         <property name="text">
          <string>17번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_18">
         <property name="text">
          <string>18번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_19">
         <property name="text">
          <string>19번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_20">
         <property name="text">
          <string>20번 PC</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="L3">
       <item>
        <widget class="QCheckBox" name="CB_21">
         <property name="text">
          <string>21번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_22">
         <property name="text">
          <string>22번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_23">
         <property name="text">
          <string>23번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_24">
         <property name="text">
          <string>24번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_25">
         <property name="text">
          <string>25번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_26">
         <property name="text">
          <string>26번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_27">
         <property name="text">
          <string>27번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_28">
         <property name="text">
          <string>28번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_29">
         <property name="text">
          <string>29번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_30">
         <property name="text">
          <string>30번 PC</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="L4">
       <item>
        <widget class="QCheckBox" name="CB_31">
         <property name="text">
          <string>31번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_32">
         <property name="text">
          <string>32번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_33">
         <property name="text">
          <string>33번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_34">
         <property name="text">
          <string>34번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_35">
         <property name="text">
          <string>35번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_36">
         <property name="text">
          <string>36번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_37">
         <property name="text">
          <string>37번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_38">
         <property name="text">
          <string>38번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_39">
         <property name="text">
          <string>39번 PC</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="CB_40">
         <property name="text">
          <string>40번 PC</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>571</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="text">
     <string>제작자 : 21학번 서동준</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_On">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>전원켜기</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="btn_Off">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>130</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>전원종료</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
